### ISSUE TITLE
trojan alert!

### ISSUE THREAD

**Issue by pesegolsson** [kyrptonaught/CMDKeybinds#60]
hi!
i got a trojan alert downloading the 20.4 version. alerting for trojan Wacatac.B!ml you might want to check your system for infection!

**Comment by Zeroxeron** [comment]
Yep, something probably is a little bit wrong here - no one of other mods triggers WinDefender
The fun fact though is that vt scan is clean -
https://www.virustotal.com/gui/file/e40ca9131061dfe048205e9b8d7151986c5daecca3fcc33b4f85cfcb83d9cc85/detection
The download was from ModRynth's page
